nixos/modules/services/web-servers/hydron.nix

```diff
@@ -43,9 +44,24 @@ in with lib; {
 
     postgresArgs = mkOption {
       type = types.str;
-      default = "user=hydron password=" + cfg.password + " dbname=hydron sslmode=disable";
-      example = "user=hydron password=dumbpass dbname=hydron sslmode=disable";
       description = "Postgresql connection arguments.";
+      default = ''
+        {
+          "driver": "postgres",
+          "connection": "user=hydron password='' + cfg.password + " " + ''dbname=hydron sslmode=disable"
+      '' + "}";
+      
+      defaultText = ''
+        {
+          "driver": "postgres",
+          "connection": "user=hydron password='' + cfg.password + " " + ''dbname=hydron sslmode=disable"
+      '' + "}";
```

Comment:
The manual will prefer `defaultText` over `default`, so using the same for both doesn't do anything. You'll want to use something like `password=\${cfg.password}` to not have Nix actually insert the password. Or use `password=${options.services.hydron.password.default}`. But on second thought, it might be better to not set the default here, but instead use `mkDefault` in the config section, such that nothing is duplicated, up to you.

And what's up with the weird quotes? I'd prefer to only use one type everywhere, you can use string interpolation `foo=${foo}` as well.

Also why is this a json string now?

Reply:
db_conf.json is a json string, so I'd rather just put the string directly, and we avoid a lot of cruft this way.
The weird quotes are just to make it look pretty on-disk, since for some reason the last `}` keeps it's tabbing and doesn't strip white space.

Reply:
Also, for some reason, the quote after cfg.password does strip whitespace, so you end up getting `password=whateverdbname=`

Reply:
Looks like string interpolation fixed my problem with the last `}` being tabbed.